src/pages/RFOX/components/Claim/ClaimSelect.tsx

```diff
@@ -0,0 +1,121 @@
+import { toBaseUnit } from 'lib/math'
+import { selectAssetById, selectFirstAccountIdByChainId } from 'state/slices/selectors'
+import { useAppSelector } from 'state/store'
+
+import type { RfoxClaimQuote } from './types'
+import { ClaimRoutePaths, type ClaimRouteProps } from './types'
+
+type ClaimRowProps = {
+  assetId: AssetId
+  amount: string
+  status: string
+  setClaimQuote: (quote: RfoxClaimQuote) => void
+}
+
+const hoverProps = { bg: 'gray.700' }
+
+const ClaimRow: FC<ClaimRowProps> = ({ assetId, amount, status, setClaimQuote }) => {
+  const translate = useTranslate()
+  const history = useHistory()
+
+  const claimAsset = useAppSelector(state => selectAssetById(state, assetId))
+  const assetSymbol = claimAsset?.symbol
+  const claimAmountCryptoBaseUnit = toBaseUnit(bnOrZero(amount), claimAsset?.precision ?? 0)
+
+  // TODO(apotheosis): make this programmatic when we implement multi-account
+  const claimAssetAccountId = useAppSelector(state =>
+    selectFirstAccountIdByChainId(state, claimAsset?.chainId ?? ''),
+  )
+
+  const claimQuote: RfoxClaimQuote = useMemo(
+    () => ({
+      claimAssetAccountId: claimAssetAccountId ?? '',
+      claimAssetId: assetId,
+      claimAmountCryptoBaseUnit,
+    }),
+    [assetId, claimAmountCryptoBaseUnit, claimAssetAccountId],
+  )
+
+  const handleClick = useCallback(() => {
+    setClaimQuote(claimQuote)
+    history.push(ClaimRoutePaths.Confirm)
+  }, [claimQuote, history, setClaimQuote])
+
+  return (
+    <Flex
+      as={Button}
+      align='center'
+      variant='unstyled'
+      p={8}
+      borderRadius='md'
+      width='100%'
+      onClick={handleClick}
+      _hover={hoverProps}
+    >
+      <Box mr={4}>
+        <AssetIconWithBadge assetId={foxAssetId}>
+          <TransactionTypeIcon type={TransferType.Receive} />
+        </AssetIconWithBadge>
+      </Box>
+      <Box mr={4}>
+        <RawText fontSize='sm' color='gray.400' align={'start'}>
+          {translate('RFOX.unstakeFrom', { assetSymbol })}
+        </RawText>
+        <RawText fontSize='xl' fontWeight='bold' color='white' align={'start'}>
+          {assetSymbol}
+        </RawText>
+      </Box>
+      <Box flex='1' alignItems={'end'}>
+        <RawText fontSize='sm' fontWeight='bold' color='green.300' align={'end'}>
+          {status}
+        </RawText>
+        <RawText fontSize='xl' fontWeight='bold' color='white' align={'end'}>
+          <Amount.Crypto value={amount} symbol={assetSymbol ?? ''} />
+        </RawText>
+      </Box>
+    </Flex>
+  )
+}
+
+type ClaimSelectProps = {
+  setClaimQuote: (quote: RfoxClaimQuote) => void
+}
+
+export const ClaimSelect: FC<ClaimSelectProps & ClaimRouteProps> = ({
+  headerComponent,
+  setClaimQuote,
+}) => {
+  return (
+    <SlideTransition>
+      <Stack>{headerComponent}</Stack>
+      <CardBody py={12}>
+        <Flex flexDir='column' gap={4}>
+          <ClaimRow
+            assetId={foxAssetId}
+            amount={'1500'}
+            status={'Available'}
```

Comment:
Just triple checking with @reallybeard we only want to show *available* claims here? If so, there is no need to have a programmatic `status` prop as they would all be available

<img width="605" alt="image" src="https://github.com/shapeshift/web/assets/17035424/f6e899c0-87d0-4691-b1b3-fef207ada7c1">